drizzlepac/hlautils/processing_utils.py

```diff
@@ -24,16 +24,22 @@
 log = logutil.create_logger(__name__, level=logutil.logging.NOTSET, stream=sys.stdout)
 
 def get_rules_file(product):
-    """Copies default HLA rules file to local directory."""
+    """Copies default HAP rules file to local directory."""
     hdu, closefits = _process_input(product)
+    rootname = '_'.join(product.split("_")[:-1])
     phdu = hdu[0].header
     instrument = phdu['instrume']
     code_dir = os.path.abspath(__file__)
     base_dir = os.path.dirname(os.path.dirname(code_dir))
-    rules_name = "{}_header_hla.rules".format(instrument.lower())
-    rules_filename = os.path.join(base_dir, 'pars', rules_name)
-    if rules_name not in os.listdir('.'):
-        shutil.copy(rules_filename, os.getcwd())
+    def_rules_name = "{}_header_hap.rules".format(instrument.lower())
+    new_rules_name = "{}_header_hap.rules".format(rootname)
+    rules_filename = os.path.join(base_dir, 'pars', def_rules_name)
+    new_rules_filename = os.path.join(os.getcwd(), new_rules_name)
+    
+    if new_rules_name not in os.listdir('.'):
+        shutil.copy(rules_filename, new_rules_filename)
+        #shutil.copy(rules_filename, os.getcwd())
+    return new_rules_name
```

Comment:
Just making sure here you want to return `new_rules_name` and not `new_rules_filename`.

Reply:
Good question!  It turns out that the 'fitsblender' package wants to look for a file with 'new_rules_name' in the current directory for use, so we do not need the full filename with path as stored in 'new_rules_filename'.

Reply:
OK.